scripts/Test-LCOW-UVM.ps1

```diff
@@ -0,0 +1,143 @@
+#ex:  .\scripts\Test-LCOW-UVM.ps1 -vb -Action Bench  -BootFilesPath C:\ContainerPlat\LinuxBootFiles\ -MountGCSTest -Count 2 -Benchtime '3s'
+# benchstat via  `go install golang.org/x/perf/cmd/benchstat@latest`
+
+[CmdletBinding()]
+param (
+    [ValidateSet('Test', 'Bench', 'List', 'Shell')]
+    [alias('a')]
+    [string]
+    $Action = 'Bench',
+
+    [string]
+    $Note = '',
+
+    # test parameters
+    [int]
+    $Count = 1,
+
+    [string]
+    $BenchTime = '5s',
+
+    [string]
+    $Timeout = '10m',
+
+    [alias('tv')]
+    [switch]
+    $TestVerbose,
+
+    [string]
+    $Run = '',
+
+    [string]
+    $CodePath = '.',
+
+    [string]
+    $OutDirectory = '.\test\results',
+
+    # uvm parameters
+    [string]
+    $BootFilesPath = 'C:\ContainerPlat\LinuxBootFiles',
+
+    [switch]
+    $DisableTimeSync,
+
+    [string]
+    $RootfsMount = '/run/rootfs',
+
+    [string]
+    $RootfsPath = (Join-Path $BootFilesPath 'rootfs.vhd'),
```

Comment:
Do you also plan to later add support for using an initrd.img instead?

Reply:
I forgot about that; yes, we should add it

Reply:
I added a flag to switch the uVM boot from vhd to initrd, but also, this flag is actually for mounting the rootfs vhd into the uVM to use as a container image for runc. Ideally it would be something like `"C:\ContainerPlatData\root\io.containerd.snapshotter.v1.windows-lcow\snapshots\2\layer.vhd"`, but I didnt want to add a dependency on using `crictl` or `ctr` to get layer paths...
I updated the variable names to clarify